Makefile

```diff
@@ -593,3 +593,15 @@ test-rosetta:
 	docker build -t rosetta-ci:latest -f client/rosetta/rosetta-ci/Dockerfile .
 	docker-compose -f client/rosetta/docker-compose.yaml --project-directory ./ up --abort-on-container-exit --exit-code-from test_rosetta --build
 .PHONY: test-rosetta
+
+##############################
+### Relayer
+##############################
+relayer-install:
+	scripts/install-relayer.sh
+
+RELAY_PATH?=local_testnet
+relayer-start: relayer-install
```

Comment:
Yeah, I purposely made it exit unless there are keys. I had to choose between letting the relayer run and hoping the user sets them up, or enforcing them to setup the keys prior. Thought it would be easier to enforce them being set up.